entente/test_cli.py

```diff
@@ -28,12 +28,9 @@ def test_transfer_landmarks_cli(tmp_path):
         assert result.exit_code == 0
 
         transferred = load_landmarks("target.json")
+
         np.testing.assert_array_equal(
```

Comment:
These aren't numpy arrays, are they? If they are identical dicts that's fine, then use `assert transferred == expected_landmarks`.

Reply:
They're each a list of dicts (ie: the new format).  So I guess that still works.